spring-cloud-kubernetes-fabric8-discovery/src/main/java/org/springframework/cloud/kubernetes/fabric8/discovery/Fabric8DiscoveryServicesAdapter.java

```diff
@@ -1,92 +0,0 @@
-/*
- * Copyright 2012-2022 the original author or authors.
```

Comment:
this was never public, and I introduced it quite recently. I added it because I was not aware of all the code-paths; but now that I am I realized that it is not needed at all